Examples/Physics_applications/laser_ion/inputs

```diff
@@ -306,6 +306,22 @@ FieldProbe_ScatLine.resolution = 201
 LBC.type = LoadBalanceCosts
 LBC.intervals = 100
 
+hist2D.type                                 = ParticleHistogram2D
+hist2D.intervals                            = 100
+hist2D.path                                 = "./"
+hist2D.species                              = electrons
+hist2D.bin_number_abs                       = 128
+hist2D.bin_number_ord                       = 256
+hist2D.bin_min_abs                          = -7.5e-6
+hist2D.bin_max_abs                          = 7.5e-6
+hist2D.bin_min_ord                          = -5.e-6
+hist2D.bin_max_ord                          =  25.e-6
+hist2D.histogram_function_abs(t,x,y,z,ux,uy,uz,w) = "x"
+hist2D.histogram_function_ord(t,x,y,z,ux,uy,uz,w) = "z"
```

Comment:
While this was a good test, we could write instead two histograms here:
- longitudinal ion phase space (z-uz)
- longitudinal electron phase space (z-uz) for electrons close (e.g., 1mu) to the laser axis `sqrt(x*x+y+y)<1e-6`